packages/kbn-expandable-flyout/README.md

```diff
@@ -9,30 +9,34 @@ The flyout is composed of 3 sections:
 - a left wider section to show more details
 - a preview section, that overlays the right section. This preview section can display multiple panels one after the other and displays a `Back` button
 
-At the moment, displaying more than one flyout within the same plugin might be complicated, unless there are in difference areas in the codebase and the contexts don't conflict with each other.
+## Design decisions
 
-## What the package offers
+The expandable-flyout package is designed to render a single flyout for an entire plugin. While displaying multiple flyouts might be feasible, it will be a bit complicated, and we recommend instead to build multiple panels, with each their own context to manage their data (for example, take a look at the Security Solution [setup](https://github.com/elastic/kibana/tree/main/x-pack/plugins/security_solution/public/flyout)).
 
-The ExpandableFlyout [React component](https://github.com/elastic/kibana/tree/main/packages/kbn-expandable-flyout/src/components/index) that renders the UI.
+The expandable-flyout is making some strict UI design decisions:
+- when in collapsed mode (i.e. when only the right/preview section is open), the flyout's width is fixed to the EUI `s` size
+- when in expanded mode (i.e. when the left section is opened), the flyout's width is fixed to the EUI `l` size 
```

Comment:
@PhilippeOberti , do you mean the whole flyout width or of a particular section. Also, how does expanded mode impacts the right section?

Reply:
thanks I added a bit more precision. I didn't want to get too much into details as most developers wouldn't care but I think I found a good compromise